File: project2/src/project2/Employee.java
package project2;

public class Employee {

	private int empId;
	private String name;
	private float salary;
	static private String companyName="Wipro";
	

	public Employee(int eid, String name, float salary) {
		
		empId = eid;
		this.name = name;
		this.salary = salary;
	}

	public Employee(int empId, String name) {
		
		this.empId = empId;
		this.name = name;
		salary=25000;
	}

	public int getEmpId() {
	
		return empId;
	}


	public String getName() {
		return name;
	}

	public float getSalary() {
		return salary;
	}
	
	public static void setCompanyName(String cn) {
		companyName=cn;
	
	}

	public void setSalary(float salary) {
		this.salary = salary;
	}

	public void show() {
		System.out.println("Emp Id: " + empId);
		System.out.println("Name: " + name);
		System.out.println("Salary: " + salary);
	}


}

File: code/employee.cpp
#include <iostream>
using namespace std;
class employee
{
public:
    int empId;
    string name;

    void setdata(int i, string n);
    void show();
    employee() : empId(0), name("unknown") {}
    employee(int eid, string nm) : empId(eid), name(nm) {}
};

void employee::setdata(int eid, string nm)
{
    empId = eid;
    name = nm;
}

void employee::show()
{
    cout << "EmpId: " << empId << endl;
    cout << "Name: " << name << endl;
}

int main()
{

    employee a, b(111, "Alok"), c;
    // a.setdata(123, "Kumar");

    a.show();
    b.show();
    c.show();
}